### PR TITLE
docs: reach: reflashing: update instructions

### PR DESCRIPTION
**Pull request by giragleb**
Update firmware reflashing guide according to the new Flasher Tool UI

### CODE REVIEW

File: docs/reach/common/reachview/firmware-reflashing.md [hunk 1]
```diff
@@ -31,88 +31,71 @@ Note that you **do not** need to do this unless you want to bring Reach to its i
...
-	* When Reach successfully reboots, you'll see "Reach is flashed and ready to use" message
-
-	<p style="text-align:center" ><img src="../img/reachview/firmware-reflashing/app-step5-flashed-reach.png" style="width: 300px;" /></p>
-
-	??? danger ""Flashing failed" error troubleshooting"
-
-		If you faced "Flashing failed" error, we recommend following the next steps:
-
-		<p style="text-align:center" ><img src="../img/reachview/firmware-reflashing/app-flashing-failed.png" style="width: 300px;" /></p>
+	* Connect Reach in Firmware mode
```
Comment (ivnsmirnov) [inline docs/reach/common/reachview/firmware-reflashing.md]: ... in Firmware **flashing** mode?

File: docs/reach/common/reachview/firmware-reflashing.md [hunk 2]
```diff
@@ -31,88 +31,71 @@ Note that you **do not** need to do this unless you want to bring Reach to its i
...
 
 		??? note "Steps for RS2 device"
 			
-			* Disconnect **Reach RS2** device from the PC
+			* Turn off the receiver using the power button
 			
-			* Hold the power button for 10 seconds until Reach turns off
+			* Press and hold the power button again and while pressing plug the USB into PC
```
Comment (ivnsmirnov) [inline docs/reach/common/reachview/firmware-reflashing.md]: @TatianaAndreeva is this step clear for you? I suggest splitting it into 3 separate steps: press and hold the button, plug the USB cable, release the button.

**Review by ivnsmirnov — COMMENTED** [review]
On Linux installation using terminal is preferred. Here are the steps:
- Press `Ctrl+Alt+T` to open terminal
- navigate to the directory with downloaded file (`cd ~/Downloads`)
- run `sudo dpkg -i reach-firmware-flash-tool_1.2.0_amd64.deb`

During the installation user will be asked for confirmation adding the current user to `plugdev`, `dialout` and `lpadmin` groups. If user selects `No`, flasher might work incorrectly.

File: docs/reach/common/reachview/firmware-reflashing.md (hunk 2 above)
Comment (TatianaAndreeva) [reply docs/reach/common/reachview/firmware-reflashing.md]: I agree

**Review by ivnsmirnov — APPROVED** [review]

**Pull request merged**